sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample08_MockingClientTypesLiveTests.cs

```diff
@@ -90,5 +91,96 @@ Task processErrorHandler(ProcessErrorEventArgs args)
 
             #endregion
         }
+
+        /// <summary>
+        ///   Performs basic smoke test validation of the contained snippet.
+        /// </summary>
+        ///
+        [Test]
+        public async Task CallingHandlersOnAnInterval()
+        {
+            #region Snippet:EventHubs_Sample08_CallingHandlersOnAnInterval
+
+            // This handler is for illustrative purposes only.
+
+            Task processEventHandler(ProcessEventArgs args)
+            {
+                // Application-defined code here
+
+                return Task.CompletedTask;
+            }
+
+            // This function simulates a random event being emitted for processing.
+
+            Random rng = new();
```

Comment:
not sure if this was a typo
```suggestion
            Random rnd = new();
```

Reply:
You got something against (r)andom (n)umber (g)enerator?   😄